From f779dc7dd62a99e74dc5d94703586d77662899e6 Mon Sep 17 00:00:00 2001
From: Moritz Warning <moritzwarning@web.de>
Date: Tue, 20 Aug 2019 23:16:14 +0200
Subject: [PATCH 02/10] config/config-build: extend small flash option

---
 config/Config-build.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/config/Config-build.in b/config/Config-build.in
index 9669fc86c7..c521006e47 100644
--- a/config/Config-build.in
+++ b/config/Config-build.in
@@ -174,6 +174,7 @@ menu "Global build settings"
 
 	config STRIP_KERNEL_EXPORTS
 		bool "Strip unnecessary exports from the kernel image"
+		default y if SMALL_FLASH
 		help
 		  Reduces kernel size by stripping unused kernel exports from the kernel
 		  image.  Note that this might make the kernel incompatible with any kernel
@@ -181,6 +182,7 @@ menu "Global build settings"
 
 	config USE_MKLIBS
 		bool "Strip unnecessary functions from libraries"
+		default y if SMALL_FLASH
 		help
 		  Reduces libraries to only those functions that are necessary for using all
 		  selected packages (including those selected as <M>).  Note that this will
-- 
2.26.2

