From 33dd582fce18a38b20756815076b7e80f5360503 Mon Sep 17 00:00:00 2001
From: Matthias Schiffer <mschiffer@universe-factory.net>
Date: Tue, 29 Dec 2015 22:48:52 +0100
Subject: [PATCH 15/33] lua: fix installation of headers for host build

---
 package/utils/lua/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/package/utils/lua/Makefile b/package/utils/lua/Makefile
index 72d5631..c37d99b 100644
--- a/package/utils/lua/Makefile
+++ b/package/utils/lua/Makefile
@@ -140,6 +140,7 @@ define Host/Install
 	$(MAKE) -C $(HOST_BUILD_DIR) \
 		INSTALL_TOP="$(STAGING_DIR_HOST)" \
 		install
+	$(CP) $(HOST_BUILD_DIR)/src/lnum_config.h $(STAGING_DIR_HOST)/include/
 endef
 
 define Build/InstallDev
-- 
2.8.0

