From 3dfc908b11b87bd6d6dba703a2fb305fa8bbaada Mon Sep 17 00:00:00 2001
From: Moritz Warning <moritzwarning@web.de>
Date: Tue, 1 Nov 2016 01:57:26 +0100
Subject: [PATCH 2/5] batctl: update to 2016.4

---
 batctl/Makefile | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/batctl/Makefile b/batctl/Makefile
index 818e345..7ae81b5 100644
--- a/batctl/Makefile
+++ b/batctl/Makefile
@@ -9,9 +9,9 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=batctl
 
-PKG_VERSION:=2016.1
-PKG_RELEASE:=1
-PKG_MD5SUM:=b98a6e3b45927315cd6d4efba4c1a1ff
+PKG_VERSION:=2016.4
+PKG_RELEASE:=0
+PKG_MD5SUM:=0112f35b2a48d32f23a17a6f45348e63
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://downloads.open-mesh.org/batman/releases/batman-adv-$(PKG_VERSION)
@@ -55,7 +55,8 @@ MAKE_BATCTL_ENV += \
         CPPFLAGS="$(TARGET_CPPFLAGS)" \
         CFLAGS="$(TARGET_CFLAGS)" \
         LDFLAGS="$(TARGET_LDFLAGS)" \
-        LIBNL_NAME="libnl-tiny"
+        LIBNL_NAME="libnl-tiny" \
+        LIBNL_GENL_NAME="libnl-tiny"
 
 MAKE_BATCTL_ARGS += \
         REVISION="$(PKG_BATCTL_SHORTREV)" \
-- 
2.10.1

