From 2b1177eaae2692ee8f310bde2ea73f75a971870c Mon Sep 17 00:00:00 2001
From: Marek Lindner <mareklindner@neomailbox.ch>
Date: Tue, 9 Jun 2015 22:12:34 +0800
Subject: [PATCH 5/5] batman-adv: upgrade package to latest release 2015.0

Signed-off-by: Marek Lindner <mareklindner@neomailbox.ch>
---
 batctl/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/batctl/Makefile b/batctl/Makefile
index 48407e9..74bb384 100644
--- a/batctl/Makefile
+++ b/batctl/Makefile
@@ -9,9 +9,9 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=batctl
 
-PKG_VERSION:=2014.4.0
+PKG_VERSION:=2015.0
 PKG_RELEASE:=1
-PKG_MD5SUM:=f3a14565699313258ee6ba3de783eb0a
+PKG_MD5SUM:=d28ad89d71888ad6b99a1f6b30c9d47c
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://downloads.open-mesh.org/batman/releases/batman-adv-$(PKG_VERSION)
-- 
2.1.4

